setup.py

```diff
@@ -113,6 +113,8 @@ def get_extras_require() -> Dict[str, List[str]]:
             "jax",
             "optax",
             "dm-haiku",
+            "hydra-core",
```

Comment:
I'm wondering `hydra-core` can be removed here, as `hydra-optuna-sweeper` explicitly depends on `hydra-core` (https://github.com/toshihikoyanase/hydra-optuna-sweeper/blob/main/setup.py#L31). What do you think?

Reply:
Either is fine. If I remove `hydra-core`, I might send another RP to remove `"jax"` line in this `setup.py`...

Reply:
Thank you for the advanced consideration. I think it's beneficial to slim a dependencies since it is becoming bigger and bigger.
As you said, `jax` could be removed because `dm-haiku` explicitly depends on it. I'd like to review your upcoming PR.